### PR TITLE
refactor(organisms): SfBottomNavigation

### PR DESCRIPTION
**Pull request by martaradziszewska**
# Scope of work
- adjusted styles
- add class 'sf-bottom-navigation-item--center' if displayed only icons or labels
- change icon sizes in stories
- add props `iconActive` and `isActive` to change on click to get active item and change icon
- updated common story [to be updated]

[IMPORTANT] 
- I think this component needs more attention to refactor
- need to replace proper icons and fix sizes
- need to add SfBadge to SfIcon

image

image

# Checklist

- [x] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed

### CODE REVIEW

File: packages/shared/styles/components/organisms/SfBottomNavigation.scss [hunk 1]
```diff
@@ -1,59 +1,80 @@
...
+  display: flex;
+  justify-content: space-around;
+  align-items: flex-end;
+  width: 100%;
+  height: var(--bottom-navigation-height, 3.75rem);
+  padding: var(--bottom-navigation-padding, 0 var(--spacer-sm));
+  background: var(--bottom-navigation-background, var(--c-white));
+  box-shadow: var(
+    --bottom-navigation-box-shadow,
+    0px -2px 10px rgba(154, 154, 154, 0.15)
```
Comment (aniamusial) [inline packages/shared/styles/components/organisms/SfBottomNavigation.scss]: aren't we sticking to the var(--c-dark) version?

File: packages/shared/styles/components/organisms/SfBottomNavigation.scss [hunk 2]
```diff
@@ -1,59 +1,80 @@
...
+
   &--floating {
-    --bottom-navigation-item-top: -1.5rem;
     --icon-color: var(--c-white);
     &:not(:hover) {
-      --circle-icon-box-shadow: 0 4px 6px rgba(66, 66, 66, 0.2);
+      --button-box-shadow: 0 4px 4px rgba(var(--c-dark-base), 0.2);
+    }
+    
+    & .sf-circle-icon {
```
Comment (aniamusial) [inline packages/shared/styles/components/organisms/SfBottomNavigation.scss]: are brackets needed here?

File: packages/vue/src/components/atoms/SfCircleIcon/SfCircleIcon.vue [hunk 3]
```diff
@@ -1,5 +1,10 @@
 <template>
-  <SfButton class="sf-circle-icon" :disabled="disabled" v-on="$listeners">
+  <SfButton
+    :aria-label="ariaLabel"
```
Comment (pspaczek) [inline packages/vue/src/components/atoms/SfCircleIcon/SfCircleIcon.vue]: @martaradziszewska we neet it? when you add on parent component it should work  on it :)

Comment (martaradziszewska) [reply packages/vue/src/components/atoms/SfCircleIcon/SfCircleIcon.vue]: it fixes accessibility violations on SfCircleIcon

File: packages/shared/styles/components/organisms/SfBottomNavigation.scss [hunk 4]
```diff
@@ -1,59 +1,80 @@
...
+
   &--has-margin {
-    --bottom-navigation-item-margin: 6px 0 0 0;
+    --bottom-navigation-item-label-margin: var(--spacer-xs) 0 0 0;
   }
+
   &--floating {
-    --bottom-navigation-item-top: -1.5rem;
     --icon-color: var(--c-white);
     &:not(:hover) {
```
Comment (pspaczek) [inline packages/shared/styles/components/organisms/SfBottomNavigation.scss]: can we use it without `&:not(:hover) {}`? maybe we should move it to SfCircleButton?

**Pull request merged**